### PR TITLE
Improve logs for ExecCondition and rescoping

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/core/cgroup.c [hunk 1]
```diff
@@ -2164,19 +2164,22 @@ int unit_attach_pids_to_cgroup(Unit *u, Set *pids, const char *suffix_path) {
                 /* First, attach the PID to the main cgroup hierarchy */
                 q = cg_attach(SYSTEMD_CGROUP_CONTROLLER, p, pid);
                 if (q < 0) {
-                        log_unit_debug_errno(u, q, "Couldn't move process " PID_FMT " to requested cgroup '%s': %m", pid, p);
+                        bool again = MANAGER_IS_USER(u->manager) && ERRNO_IS_PRIVILEGE(q);
 
-                        if (MANAGER_IS_USER(u->manager) && ERRNO_IS_PRIVILEGE(q)) {
+                        log_unit_full_errno(u, again ? LOG_DEBUG : LOG_INFO,  q,
+                                            "Couldn't move process " PID_FMT " to requested cgroup '%s': %m", pid, p);
```
Comment (poettering) [inline src/core/cgroup.c]: maybe if again is true change the log message to include "directly" here, to make things clearer what this precise message is about, i.e. about the direct attempt. and the indirect one follows right below then.

**Comment by poettering** [comment]
looks good to me. one trivial suggestion. too ttirival too even change the label to "minor-suggestion"... ;-)

**Comment by keszybz** [comment]
Updated with "directly". CI passed before, so I guess we can merge this as soon as some CI passes.

**Comment by stupicode** [comment]
**journalctl -p 3 -b:** 
Jul 24 12:35:44 asus systemd[2120]: Failed to start Application launched by gnome-session-binary.

**gnome-shell --version:**
GNOME Shell 40.3

**echo $XDG_SESSION_TYPE**
x11

**uname -a**
Linux asus 5.12.15-arch1-1

**Pull request merged**